jenkins_job_wrecker/cli.py

```diff
@@ -27,6 +28,7 @@
 
 
 def str_presenter(dumper, data):
+    data = replace_tab(data)
     if len(data.splitlines()) > 1:  # check for multiline string
```

Comment:
maybe it's worth to put `data = replace_tab(data)` under this condition to avoid splitting 3 times in the worst case? (twice just to make sure it's multiline)

Reply:
I have thought about this, but I have to replace tabs even if it's not multiline.